d2/src/main/java/com/linkedin/d2/jmx/ZooKeeperAnnouncerJmxMXBean.java

```diff
@@ -47,6 +47,13 @@
    */
   void changeWeight(boolean doNotSlowStart) throws PropertyStoreException;
 
+  /**
+   * Set {@link com.linkedin.d2.balancer.properties.PropertyKeys#DO_NOT_LOAD_BALANCE} for a given uri.
+   *
+   * @param doNotLoadBalance Flag to let clients know if load balancing should be disabled for a host.
+   */
+  void doNotLoadBalance(boolean doNotLoadBalance) throws PropertyStoreException;
```

Comment:
Or setDoNotLoadBalance

Reply:
sure will rename to setDoNotLoadBalance. I'd rather keep the explicit "do not" to make it clear this is not default behavior. 

The method at line 48 is different because it's changing the weight, not only disabling/enabling slow start.